jsapp/js/account/plan.module.scss

```diff
@@ -0,0 +1,224 @@
+@use 'scss/sizes';
+@use '~kobo-common/src/styles/colors';
+@use 'scss/breakpoints';
+@use 'sass:color';
+@use "scss/_variables";
+
+.accountPlan {
+  padding: sizes.$x30 sizes.$x40;
+  overflow-y: auto;
+  position: relative;
+  height: 100%;
+  width: 100%;
+  font-family: "Roboto", sans-serif;
+}
+
+.plansSection {
+  display: grid;
+  grid-template-rows: repeat(4, auto);
+  grid-template-columns: repeat(auto, minmax(300px, 1fr));
+  column-gap: sizes.$x20;
+  margin-bottom: sizes.$x16;
+}
+
+.intervalToggle {
+  grid-column: 1 / 5;
+  grid-row: 1/2;
+  justify-self: center;
+  background-color: colors.$kobo-cloud;
+  padding: sizes.$x3;
+  border-radius: sizes.$x4;
+  margin-bottom: sizes.$x20;
+}
+
+.intervalToggle input[type="radio"] {
+  display: none;
+  position: absolute;
+  width: 100%;
+  appearance: none;
+}
+
+.intervalToggle label {
+  color: colors.$kobo-dark-blue;
+  padding: 0 sizes.$x12;
+  font-size: sizes.$x18;
+  line-height: sizes.$x32;
+  display: inline-block;
+  width: sizes.$x100;
+  text-align: center;
+  cursor: pointer;
+}
+
+.intervalToggle input[type="radio"]:checked+label {
+  background-color: colors.$kobo-blue;
+  color: colors.$kobo-white;
+  border-radius: sizes.$x4;
+  width: sizes.$x100;
+  text-align: center;
+}
+
+.intervalToggle input[type="radio"]:checked+label:after {
+  background-color: colors.$kobo-blue;
+  padding: sizes.$x8 sizes.$x12;
+}
+
+.intervalToggle input[type="radio"]:checked+label:before {
+  width: sizes.$x20;
+  height: sizes.$x20;
+  color: colors.$kobo-cloud;
+  padding: sizes.$x8 sizes.$x12;
+}
+
+.intervalToggle input[type="radio"]:checked+label:hover{
+  background-color: color.adjust(colors.$kobo-blue, $lightness: -5%);
+}
+
+.intervalToggle input[type="radio"]+label:hover{
+  background-color: color.adjust(colors.$kobo-cloud, $lightness: -5%);
+  border-radius: sizes.$x4;
+}
+
+.planContainer {
+  grid-auto-columns: 1fr;
+  width: 25em;
+  grid-row: 3/4;
+  border-radius: sizes.$x6;
+  border: colors.$kobo-gray-85 solid 1px;
+  padding: sizes.$x36 sizes.$x48 sizes.$x40;
+}
+
+.priceName,
+.enterpriseTitle {
+  color: colors.$kobo-gray-40;
+  font-weight: 700;
+  font-size: sizes.$x24;
+  line-height: sizes.$x40;
+  margin: 0;
+  padding:0;
+}
+
+.priceName {
+  text-align: center;
+}
+
+.enterpriseTitle {
+  margin: 0 0 sizes.$x16;
+}
+
+.priceTitle {
+  color: colors.$kobo-dark-blue;
+  text-align: center;
+  font-size: sizes.$x18;
+  padding-bottom: sizes.$x16;
+  font-weight: 600;
+  line-height: sizes.$x20;
+}
+
+.btnPlacement {
+  margin: 0 auto;
+}
+
+.planContainer > Button {
+  margin: sizes.$x20 auto 0;
+}
+
+.btnSpacePlaceholder { 
+  margin: sizes.$x20 0;
+  height: sizes.$x32;
+}
+
+.planContainer :global hr {
+  border-bottom: 1px solid colors.$kobo-gray-85;
+  border-top: 1px solid colors.$kobo-gray-85;
```

Comment:
<img width="327" alt="Screenshot 2023-04-03 at 12 06 59" src="https://user-images.githubusercontent.com/2521888/229494458-b2f845d3-78da-4b8f-b9ad-
f420ad49f78d.png">

This line should be `1px` height, now it is rendered as `2px` :)